### PR TITLE
Functional test Digital Exhibits and Collections page

### PR DESCRIPTION
**Pull request by SSJohns**
A few test cases for the collections pages. This is the [spreadsheet](https://docs.google.com/spreadsheets/d/1U1JFZprPO4cTgxBvM7xCpGzoXJk7fD25aU0epXohmL4/edit?ts=58d03245#gid=248186373) it came from.

### CODE REVIEW

File: spec/dec/functional/func_dec_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    visit '/'
+    home_page = Dec::Pages::HomePage.new
+    expect(home_page).to be_on_page
+  end
+
+  scenario 'DEC_2', :read_only do
+    visit '/'
+    home_page = Dec::Pages::HomePage.new
+    expect(home_page).to be_on_page
+    first('span', {text: 'Explore', visible: false}).click
```
Comment (jeremyf) [inline spec/dec/functional/func_dec_spec.rb]: I'm seeing duplication of this behavior. Extract the behavior to a method on the HomePage object.

File: spec/dec/functional/func_dec_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
+  scenario 'DEC_3', :read_only do
+    visit '/'
+    home_page = Dec::Pages::HomePage.new
+    first('span', {text: 'Explore', visible: false}).click
+    collections_page = Dec::Pages::CollectionsPage.new
+    sleep(5)
+    expect(collections_page).to be_on_page
+    first('span', {text: 'arrow_forward'}).trigger('click')
```
Comment (jeremyf) [inline spec/dec/functional/func_dec_spec.rb]: What is this attempting to accomplish? Extract the behavior to the CollectionsPage and provide a meaningful method name.

File: spec/dec/pages/home_page.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module Dec
+  module Pages
+    class HomePage
+      include Capybara::DSL
+      include CapybaraErrorIntel::DSL
+
+      def on_page?
+        on_valid_url? &&
+          # status_response_ok? &&
```
Comment (jeremyf) [inline spec/dec/pages/home_page.rb]: Remove commented out code.

File: spec/dec/pages/home_page.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        current_url == File.join(Capybara.app_host)
+      end
+
+      def status_response_ok?
+        status_code == 200
+      end
+
+      def valid_page_content?
+        within('.collectionscover') do
+          has_css?('span', :text => 'Digital Collections')
```
Comment (jeremyf) [inline spec/dec/pages/home_page.rb]: Replace the old Hash rocket syntax (eg. `:text =>`) with `text: `. Likewise in cases below as well.

File: spec/dec/functional/func_dec_spec.rb (hunk 2 above)
Comment (SSJohns) [inline spec/dec/functional/func_dec_spec.rb]: Part of the scenario was to click the forward arrow.

**Pull request merged**